<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesDialog</class>
 <widget class="QDialog" name="NotesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>811</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>811</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>记事本</string>
  </property>
  <widget class="QTableView" name="list_notes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>411</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="box_select">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>查询</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_condition">
         <property name="text">
          <string>查询条件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_condition"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_select_content">
         <property name="text">
          <string>查询内容：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_select_content">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_summary">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>事件概要</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_writer">
         <property name="text">
          <string>记录人：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_writer">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_unit">
         <property name="text">
          <string>记录人所在单位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_unit">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>501</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,8">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>事件标题：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_title">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_date">
         <property name="text">
          <string>记录日期：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date_tm">
         <property name="date">
          <date>
           <year>2017</year>
           <month>6</month>
           <day>8</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_type">
         <property name="text">
          <string>事件类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_type">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_content">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>521</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>事件内容</string>
   </property>
   <widget class="QPlainTextEdit" name="edit_content">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>NotesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
